webpack.config.js

```diff
@@ -0,0 +1,170 @@
+import webpack from 'webpack';
+import {CleanWebpackPlugin} from 'clean-webpack-plugin';
+import VirtualModulesPlugin from 'webpack-virtual-modules';
+import CopyPlugin from 'copy-webpack-plugin';
+import MiniCssExtractPlugin from 'mini-css-extract-plugin';
+import RemovePlugin from 'remove-files-webpack-plugin';
+import fs from 'fs/promises';
+import path from 'path';
+import {createRequire} from 'module';
+import globPkg from 'glob';
+import TerserPlugin from 'terser-webpack-plugin';
+import postcssUrl from 'postcss-url';
+import got from 'got';
+import CssMinimizerPlugin from 'css-minimizer-webpack-plugin';
+
+const git = createRequire(import.meta.url)('git-rev-sync');

```

Comment:
why is this needed instead of a normal import?

Reply:
Some ES modules are not able to be imported in Node 14+, you find the solution to this [here](https://dev.to/jimthedev/typeerror-cannot-read-property-filename-of-undefined-in-es-modules-in-node-14-and-greater-6ia)